.github/workflows/dax.yml

```diff
@@ -68,3 +90,7 @@ jobs:
           UMF_TESTS_DEVDAX_PATH="/dev/dax${UMF_TESTS_DEVDAX_NAMESPACE}"
           UMF_TESTS_DEVDAX_SIZE="$(ndctl list --namespace=namespace${UMF_TESTS_DEVDAX_NAMESPACE} | grep size | cut -d':' -f2 | cut -d',' -f1)"
           ctest -C ${{matrix.build_type}} -R devdax -V
+
+      - name: Run only tests of the file memory provider with FSDAX
```

Comment:
can't it be simple named `Run FSDAX tests`...? the name is pretty long

Reply:
Done

Reply:
thx!